alongtheway.txt

```diff
@@ -534,9 +534,9 @@ Sometimes the junk in our life is a result of reaping what we have sown but some
 
 Day 3: Mary's journey to Bethlehem
 Read: Luke 2:1-21 especially verses 4-7
-Try to keep yesterday's reading in mind. Look back at what you read and wrote down about what the angel Gabriel said about who Jesus would be.
+Try to keep yesterday's reading in mind. Look back at what you read and wrote concerning the angel Gabriel's words about who Jesus would be.
 
-Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than in a barn, in a strange place, away from all of my family and friends.
+Reflect: When I stop to think about it, and try to put myself in Mary’s position, this story is going pretty much exactly the opposite of how I would have thought things would go. If an angel appeared to me and told me I was highly favored and carrying the son of God, savior of the world, personally, I would expect to be birthing him somewhere other than a strange place, in a barn, away from all of my family and friends.
```

Comment:
ok. we'll stick with what you have. Welp, I think that's it for the revisions that I had. I'm going to plop this back into Microsoft word and start formatting. I'll keep a close eye for all the emphasized words and phrases to make sure they get added back in. Thanks!